kousa/lib/beef/schemas/user.ex

```diff
@@ -77,6 +78,7 @@ defmodule Beef.Schemas.User do
     field(:avatarUrl, :string)
     field(:bannerUrl, :string)
     field(:bio, :string, default: "")
+    field(:isBot, :boolean, default: false)
```

Comment:
we don't want this, we already have `botOwnerId` and we can track if a user is a bot or not by using that

Reply:
thanks for noticing, will fix

Reply:
So if I understand correctly, botOwnerId (in `kousa/lib/beef/schemas/user.ex`) is either `null/nil` for regular accounts or an `Ecto` type for bots?

Reply:
also we probably don't wanna do this at the backend, this can maybe be done fully on the client side